packages/core/popper/src/popper.ts

```diff
@@ -359,7 +359,7 @@ const UNMEASURED_POPPER_STYLES: CSS.Properties = {
   top: 0,
   left: 0,
   opacity: 0,
-  pointerEvents: 'none',
+  transform: 'translate3d(0, -200%, 0)',
```

Comment:
Whilst writing the recap in the issue description, I'm still wondering whether it makes more sense to actually apply the `pointerEvents` properly as we actually have what it takes to do so, and it seems like a more precise fix for the situation described.

It's that different from what we already do there:
https://github.com/radix-ui/primitives/blob/main/packages/react/popper/src/Popper.tsx#L100-L104

Reply:
It seems we created a conflict of issues because the element is on the page during measuring phase when it shouldn't be. If it's off the page, it can't get pointer events so `pointerEvents: none` wouldn't really matter. 

I'm happy with either approach tho tbh 🙂

Reply:
Yeah I suppose that's true haha
So I guess from that standpoint this fix is just as legitimate then. Let's roll with it.